doc/source/overview_File_Dataset_Container.rst

```diff
@@ -2,24 +2,22 @@
 
-Files can be grouped into datasets (a named set of files) and datasets can be grouped into containers (a named set of datasets or, recursively, containers). All three types of names refer to data so the term ‘data identifier set’ (DIS) is used to mean any set of file, dataset or container identifiers. A data identifier is just the name of a single file, dataset or container.
-
-The figure below gives an example of an aggregation hierarchy:
+Files can be grouped into datasets (a named set of files) and datasets can be grouped into containers (a named set of datasets or, recursively, containers). All three types of names refer to data so the term ‘data identifier’ (DID) is used to mean any set of file, dataset or container identifiers. A data identifier is just the name of a single file, dataset or container.
 
 
 **************************
 Data identifiers and scope
 **************************
 
-Files, datasets and containers follow an identical naming scheme which is composed of two strings: the scope and a name. The combination of both is called a data identifier (DI). For instance a file identifier (LFN) is composed of a scope and a file name.
+Files, datasets and containers follow an identical naming scheme which is composed of two strings: the scope and a name. The combination of both is called a data identifier (DID). Thus for files, the Logical File Name (LFN), a term commonly used in DataGrid terminology to identify files is equivalent to the DID in Rucio.
 
-The scope string partitions the name space in several sub spaces. The primary use case for this is to have separate scopes for production and individual users.
+The scope string partitions the name space into several sub namespaces. The primary use case for this is to easily separate centrally created data from individual user data.
 
-By default accounts will have read access to all scopes and write access only to their own scope. Privileged accounts will have write access to multiple scopes, e.g., production might use scopes such as mc11, data12_8TeV, tmp.prod.#
+By default accounts have read access to all scopes and write access only to their own scope. Privileged accounts have write access to multiple scopes, e.g., the Workload Management System is allowed to write into official data scopes.#
 
-Files, datasets and containers are uniquely identified over all time. This implies that an identifier, once used, can never be reused to refer to anything else at all, not even if the data it referred to has been deleted from the system.
+Files, datasets and containers are uniquely identified over all time. This implies that a data identifier, once used, can never be reused to refer to anything else at all, not even if the data it referred to has been deleted from the system.
 
```

Comment:
do we have concrete examples in this section ?

Reply:
No, we did have some dataset/container hierarchy image at some point, but I can't find it anymore in git history...